docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -3,8 +3,8 @@ title: 'Rules'
 lang: en
 ---
 
-## ApexFlsViolationRule
-ApexFlsViolationRule detects [Create, Read, Update, and Delete (CRUD) and Field-Level Security (FLS) violations](https://www.youtube.com/watch?v=1ZYjpjPTIn8). To invoke ApexFlsViolationRule, run `scanner:run:dfa`.
+## ApexFlsViolationRule#
+ApexFlsViolationRule detects [Create, Read, Update, and Delete (CRUD) and Field-Level Security (FLS) violations](https://www.youtube.com/watch?v=1ZYjpjPTIn8). To run the data flow analysis rules–ApexFlsViolationRule and ApexNullPointerExceptionRule–run `scanner:run:dfa`. Alternatively, run `scanner:run:dfa --category “Security”` to run only the ApexFlsViolationRule.
```

Comment:
Missed this initially. A comment about formatting that I [added for another line](https://github.com/forcedotcom/sfdx-scanner/pull/1034/files#r1148063206) applies here as well.
"data flow analysis rules" ===> "Path-based Analysis rules"

Full command:
```
sfdx scanner:run:dfa --category "Security" --projectdir /project/dir --target /project/dir/target
```